server/src/main/java/io/druid/segment/realtime/plumber/Sink.java

```diff
@@ -173,6 +179,8 @@ public boolean isEmpty()
     }
   }
 
+  //add methods for byte mem checks
```

Comment:
This looks like a stray comment that you meant to delete?

Reply:
oops, yes.